[Ruby] Implement new Concept Exercise: conditionals-ternary #1903 (#2310)
exercism/v3: languages/ruby/exercises/concept/conditionals-ternary/.docs/instructions.md

> In this exercise you will rewrite `if/else` statements from a movie theater's website into ternary conditionals.

Reviewer: This is a great premise! You could elaborate on it more if you wanted.

Also, rather than asking students to rewrite the expression, just ask them to write the expression?

Reviewer 2: I like the premise of working on a movie theatre website. The "rewrite `if/else` statements" bit seems like it may be more suited for the actual tasks (see [this suggestion](https://github.com/exercism/v3/pull/2310/files#r502860694)). Maybe something like this:

```suggestion
In this exercise you've been hired to build your local movie theater's website. As you've just graduated, you're determined to write the cleanest, most concise code you can.
```

## 1. Check if a moviegoer is entitled to the seniors' discount

> Rewrite the `Moviegoer#ticket_price` method to utilize the ternary operator.

Reviewer: ```suggestion
Tickets at the movie theatre are $15 dollars.  However, if a person is 60 years of age or older, the ticket prices is discounted to $10.  Write the `Moviegoer#ticket_price` method to return the price of the ticket based on the movie goers' age utilizing the ternary operator.
```

Reviewer 2: I really like the suggestion. A similar description can be used for the other tasks, especially with the "utilizing the ternary operator" bit.

```ruby

>   def ticket_price
>     if age < 60
>       15
>     else
>       10
>     end
>   end

Reviewer: Rather than the re-phrased answer here, maybe just put an example usage with the output:

```suggestion
new Moviegoer(20, false).ticket_price
# => 15
new Moviegoer(77, false).ticket_price
# => 10
```

```

## 2. Check if a moviegoer is allowed to see scary movies

> 

Reviewer: Write a similar description of the task to accomplish.

```ruby
  def watch_scary_movie
    if age >= 18
      "Enjoy the movie!"
    else
      "You must be over 18 to see this movie."
    end

>   end

Reviewer: Same as above

```

## 3. Check if a moviegoer is entitled to free popcorn 

> 

Reviewer: Same as above, write a descriptive summary of the task.

```ruby
  def claim_free_popcorn
    if movie_club_member
      "Enjoy your free popcorn!"
    else
      "Join the Movie Club to get free popcorn."
    end

>   end

Reviewer: Same as above
